sim.yml

```diff
@@ -0,0 +1,62 @@
+version: "3.3"
+services:
+
+  mavlink2rest:
+    image: ericjohnson97/iq_sim_mavlink2rest:t0.11.19
+    network_mode: host
+    environment:
+      - MAVLINK_SRC=udpin:127.0.0.1:14551
+      - SERVER_PORT=0.0.0.0:6040
+
+  mavp2p:
+    image: ericjohnson97/iq_sim_server_mavp2p:latest
+    network_mode: "host"
+    environment:
+      CMD: "./mavp2p tcpc:127.0.0.1:5760 udpc:127.0.0.1:14550 udpc:127.0.0.1:14551"
+    restart: on-failure
+
+
+  # vehicle simulators: uncomment the one you want to use
```

Comment:
Docker profiles are probably better than comment blocks
https://docs.docker.com/compose/profiles/

Reply:
I did not know about compose profiles. That's pretty nifty! I'll rewrite

Reply:
updated. I learned something new today. Very cool!